src/ethereum_test_forks/forks/forks.py

```diff
@@ -961,16 +976,6 @@ def engine_new_payload_version(
         """
         return 4
 
-    @classmethod
-    def engine_forkchoice_updated_version(
```

Comment:
This is also going to affect `execute hive` since the `max_blobs_per_block` has to be included when a new payload is being requested via the attributes in `engine_forkchoiceUpdatedV4`.

Reply:
https://github.com/ethereum/execution-spec-tests/issues/958

Created an issue for this and will follow up in new PR.

Reply:
I think I fixed the `execute hive` part here: [3c7ca39](https://github.com/ethereum/execution-spec-tests/pull/931/commits/3c7ca3979dbff6ec746d7c38a96acf72548e46dd)